v2/spanner-common/terraform/samples/mysql-vpc-spanner/README.md

```diff
@@ -0,0 +1,301 @@
+## Sample Scenario: MySQL in a VPC with Spanner
+
+> **_SCENARIO:_** This Terraform example illustrates launching a MySQL 5.7
```

Comment:
5.7 will be EOL soon - why we chose this?

Reply:
I can try for MySQL 8 but I didn't pay attention to the version that much to be honest since this was to illustrate a customer environment setup more than anything else. 

Will spend some time to see which I can this work for MySQL 8.

Reply:
Added examples of spinning up both MySQL 5.7 and 8.4.